client/src/components/scoreStepsList.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {GridContainer, Grid} from '@trussworks/react-uswds';
+
+const ScoreStepsList = () => {
```

Comment:
Is the TimeLinePage in the `pages` directory  related to this effort or is that something separate?